pkg/manifest/manifest.go

```diff
@@ -151,6 +151,15 @@ func (m *Manifest) ApplyDefaults() error {
 			m.Services[i].Health.Path = "/"
 		}
 
+		if s.Health.Port.Port == 0 && s.Port.Port > 0 {
+			m.Services[i].Health.Port.Port = s.Port.Port
+			if s.Port.Scheme != "" {
+				m.Services[i].Health.Port.Scheme = s.Port.Scheme
+			} else {
+				m.Services[i].Health.Port.Scheme = "http"
+			}
+		}
+
```

Comment:
What will happen if health port is defined but health scheme is not defined?

Reply:
As the code shows, if you don't define the scheme, it will default to http.

Reply:
I am asking when **health port is defined** case. i can  see in the code if health port defined the it will not execute the scheme condition in here

Reply:
Oh, I see, sorry I misunderstood 😆 Yeah, I may have missed that... let me update!

Reply:
@nightfury1204 added default now, thank you!